src/test/kotlin/ractingCar/entity/CarTest.kt

```diff
@@ -0,0 +1,47 @@
+package ractingCar.entity
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+
+internal class CarTest {
+
+    @Test
+    fun `play 함수에 0을 넣으면 움직이지 않는다`() {
+        // given
+        val car = Car()
+
+        // when
+        car.move()
+
+        // then
+        Assertions.assertThat(car.moves).isGreaterThanOrEqualTo(0)
```

Comment:
```suggestion
        // then
        Assertions.assertThat(car.moves).isEqualTo(0)
```
테스트는 움직이지 않는다인데, 실제로 움직여도 테스트는 통과하게 되어있네요.
isEqualTo로 변경해야할 것 같네요.